scripts/prep_local_devel_env.sh

```diff
@@ -159,9 +159,20 @@ while [ "$?" -ne 1 ]; do
   oc get pods -n ${ASB_PROJECT} | grep asb
 done
 
+oc scale deployment etcd --replicas 0 -n ${ASB_PROJECT}
```

Comment:
That's fine, but what about the loop below ```oc get pods -n ${ASB_PROJECT} | grep etcd```.  If you ran catasb first then there is not etcd here.